app/v1/models/errors/mtdErrors.scala

```diff
@@ -157,7 +157,7 @@ object RuleGiftAidNonUkAmountWithoutNamesError
 
 object RuleGiftsNonUkAmountWithoutNamesError
     extends MtdError(
-      code = "RULE_GIFTS_NON_UK_AMOUNT_WITHOUT_NAMES",
+      code = "RULE_GIFTS_NON_UK_INVESTMENTS_AMOUNT_WITHOUT_NAMES",
```

Comment:
RULE_GIFTS_NON_UK_AMOUNT_WITHOUT_NAMES is the error mapped to this message, I can see it was that beforehand?